src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/UnknownLinesHandler.kt

```diff
@@ -134,9 +134,12 @@ object UnknownLinesHandler {
             SbPattern.carnivalCatchStreakPattern,
             SbPattern.carnivalAccuracyPattern,
             SbPattern.carnivalKillsPattern,
-            *remoteOnlyPatterns,
         )
 
+        if (!::remoteOnlyPatterns.isInitialized) {
```

Comment:
I read this code as "if the list does not yet contain data, fill the other list with its data". this sounds wrong. is the `!` unnecessary here?

Reply:
You're right, good catch